Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+node('Slave-Azure'){
```

Comment:
Not sure you want to add this jenkins file with credentials...

Reply:
I made another pull request with all fixed. #414

Reply:
Due some mistakes, I made another pull request with all fixed. #414